contracts/std-lib-extended/src/bytes.sw

```diff
@@ -33,7 +33,7 @@ impl EvmAddress {
     /// Gets an EvmAddress from a pointer to packed bytes.
     fn from_packed_bytes(ptr: raw_ptr) -> Self {
         // The EvmAddress value will be written to this b256.
-        let value: b256 = ZERO_B256;
+        let mut value: b256 = ZERO_B256;
```

Comment:
I made this change - tests were failing here after a recent change in the compiler bc we were trying to modify a non-mut value